src/Application/Messages/Commands/Common/BaseModifyMessageCommand.cs

```diff
@@ -1,6 +1,13 @@
 ﻿namespace CoduTeam.Application.Messages.Commands.Common;
 
-public abstract record BaseModifyMessageCommand(string Content)
+public abstract record BaseModifyMessageCommand
 {
-
+    protected BaseModifyMessageCommand(string Content, DateTime SentTime)
+    {
+        this.Content = Content;
+        this.SentTime = DateTime.UtcNow;
```

Comment:
And for future) It's better to use DateTime throught provider) Try to find one in solution